src/components/layout.tsx

```diff
@@ -0,0 +1,15 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Flex } from "strict-ui"
```

Comment:
Did we import strict ui? Not seeing it.

Reply:
Oh, good point! I switched away and missed this one.